<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 25, 30)</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Calibri Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(90, 90, 90);</string>
         </property>
         <property name="text">
          <string>Timers:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="font">
          <font>
           <family>Calibri Light</family>
           <pointsize>14</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(190, 190, 190);
font: 25 14pt &quot;Calibri Light&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="delete_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(20, 20, 20);
background-color: rgb(255, 105, 105, 180);
border-radius: 25px;
padding: 15px;
min-width: 5em;</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="add_button">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(20, 20, 20);
background-color: rgba(0, 200, 220, 180);
border-radius: 25px;
padding: 15px;
min-width: 5em;</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="timer_layout">
         <item alignment="Qt::AlignTop">
          <widget class="QLabel" name="Timer_name">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>20</pointsize>
             <weight>3</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(190, 190, 190);
font: 25 20pt &quot;Calibri&quot;</string>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QLabel" name="Timer_mode">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Calibri Light</family>
             <pointsize>14</pointsize>
             <weight>3</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(90, 90, 90);
font: 25 14pt &quot;Calibri Light&quot;;</string>
           </property>
           <property name="text">
            <string>mode</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Timer_time">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>30</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(190, 190, 190);
font: 75 30pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>00:00:00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignBottom">
          <widget class="QProgressBar" name="progressBar">
           <property name="minimumSize">
            <size>
             <width>370</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
}

QProgressBar::chunk {
    background-color: #05B8CC;
	width: 1px;
 }</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="note_layout">
         <item alignment="Qt::AlignTop">
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>20</pointsize>
             <weight>3</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(190, 190, 190);
font: 25 20pt &quot;Calibri&quot;</string>
           </property>
           <property name="text">
            <string>Note:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QLabel" name="Timer_note">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Calibri Light</family>
             <pointsize>16</pointsize>
             <weight>3</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="toolTipDuration">
            <number>-1</number>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(190, 190, 190);
font: 25 16pt &quot;Calibri Light&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttons_layout">
         <item>
          <widget class="QPushButton" name="stop_button">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:rgb(20, 20, 20);
background-color: rgb(255, 105, 105, 180);
border-radius: 25px;
padding: 15px;
min-width: 5em;
}

QPushButton:pressed {
    background-color: rgb(255, 105, 105, 150);
}</string>
           </property>
           <property name="text">
            <string>STOP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="play_button">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:rgb(20, 20, 20);
background-color: rgb(5, 184, 204);
border-radius: 55px;
padding: 15px;
min-width: 5em;
min-height: 5em;
}

QPushButton:pressed {
    background-color: rgb(5, 184, 204, 180);
}</string>
           </property>
           <property name="text">
            <string>START</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset_button">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:rgb(20, 20, 20);
background-color: rgb(190, 190, 190, 180);
border-radius: 25px;
padding: 15px;
min-width: 5em;
}

QPushButton:pressed {
    background-color: rgb(190, 190, 190, 150);
}</string>
           </property>
           <property name="text">
            <string>RESET</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
